### DISCUSSION TITLE
Help with Informix engine

### DISCUSSION THREAD

**Discussion by joseajunior** [sqlalchemy/sqlalchemy#10228] [Usage Questions]
I need help doing some connections to an Informix database with version 2.0.

I tried basically every dialect to install and each of them gave me a different error.

Some of the libraries I tried: sqlalchemy-informix, ibm_db, ibm_db_sa, pyobdc, IfxPy... any of them worked.

But with the version < 2 I'm able to connect using the sqlalchemy-informix.

Does any of you uses informix databases with the latest version?

P.S.: (I'm not sure but...) DB2 databases are not the same as this informix databases I'm trying to connect.

**Comment by CaselIT** [comment]
Hi,

you will probably have better support if you report you issue to the `sqlalchemy-informix` repo (even if does not seem particularly maintained).

**Reply by joseajunior** [reply]
Maybe, but is not so much about the sqlalchemy-informix lib but how I can connect to an Informix database using the latest version of sqlalchemy.
And can be with any "dialect" as long as it works.

**Reply by zzzeek** [reply]
you would need....an informix dialect.

**Reply by joseajunior** [reply]
yeah, but which one? In my comment, I said I tried several different libs to connect and any of them worked.

**Reply by zzzeek** [reply]
it's very likely that none of them have been updated to work with SQLAlchemy 2.0 so if you have older versions working, you'd have to stick with that until someone updates the dialects

**Reply by joseajunior** [reply]
Yeah, I was thinking that too, but I thought would be good asking here to see if someone else had the same issue and would help me.

**Reply by CaselIT** [reply]
if the dialects work in 1.4 it should not be too difficult to adapt them for v2. From memory no major mandatory change was made in the update 1.4 -> 2.0 on the dialect side

**Comment by joseajunior** [comment]
I was able to connect using ibm_db_sa, don't even know what I did hahaha.
I'm closing this one.
